res/values/strings.xml

```diff
@@ -125,9 +124,10 @@
 
     <string name="message_delivered">Message delivered</string>
     <string name="message_read">Message read</string>
-
+    <string name="message_not_verified">The message was sent with non-verified encryption. See \"Info\" for details.</string>
```

Comment:
this message comes from a hack in https://github.com/deltachat/deltachat-desktop/blob/master/src/main/deltachat/chatlist.js#L110 - there are good chances that the original string will change.

Reply:
i filed an issue for that, https://github.com/deltachat/deltachat-desktop/issues/1319 , so maybe remove the string for now.

Reply:
Should I remove this?